<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TT_DialogEdit</class>
 <widget class="QDialog" name="TT_DialogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit attributes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbX">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leX"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbY">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leY"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbHasZ">
       <property name="text">
        <string>Has a Z value</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="cbHasZ">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbZ">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leZ"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbIh">
       <property name="text">
        <string>Instrument height</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leIh"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbV0">
       <property name="text">
        <string>Start angle (v0)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="leV0"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lbPh">
       <property name="text">
        <string>Prism height</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lePh"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lbHa">
       <property name="text">
        <string>Horizontal angle</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="leHa"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lbVa">
       <property name="text">
        <string>Vertical angle</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="leVa"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lbId">
       <property name="text">
        <string>Inclined distance</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="leId"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TT_DialogEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TT_DialogEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
